nsupdate/main/forms.py

```diff
@@ -42,6 +44,18 @@ class Meta(object):
 
 
 class CreateDomainForm(forms.ModelForm):
+    def clean_nameserver_update_secret(self):
+        secret = self.cleaned_data['nameserver_update_secret']
+        try:
+            binascii.a2b_base64(secret.encode(encoding="ascii", errors="strict"))
+        except (binascii.Error, UnicodeEncodeError):
+            raise forms.ValidationError(
+                _("Enter a valid Base64 secret."),
+                code='invalid'
+            )
+
```

Comment:
code: 1 line, no empty line afterwards.

why is it called clean_nameserver_update_secret? isn't it rather validate_.......?